content/slides/slide03.md

```diff
@@ -3,6 +3,6 @@ title: "Virtual Slides"
 date: 2021-02-17
 type: slides
 weight:
-slideId: 03
+slideId: '03'
```

Comment:
Is this an issue with any of the other slides numbered < 10?

Reply:
Yes all the ones with zeroes in front. I just want to make sure this is a good fix before making the others strings as well

Reply:
got it.